atomicapp/providers/kubernetes.py

```diff
@@ -204,3 +205,52 @@ def undeploy(self):
             if self.config_file:
                 cmd.append("--kubeconfig=%s" % self.config_file)
             self._call(cmd)
+
+    def persistent_storage(self, graph, action):
+        """
+        Actions are either: run, stop or uninstall as per the Requirements class
+        Curently run is the only function implemented for k8s persistent storage
+        """
+
+        logger.debug("Persistent storage enabled! Running action: %s" % action)
+
+        if action is "run":
+            pass
+        else:
+            logger.warning(
+                "%s action is not available for provider %s. Doing nothing." %
+                (action, self.key))
+            return
+
```

Comment:
Would rather do:
```
if action not in ['run']:
    logger.warning()
```

In the future when we add more it would be:
```
if action not in ['run', 'stop', etc..]:
    logger.warning()
```